server/src/main/java/com/microsoft/java/bs/core/internal/managers/BuildTargetManager.java

```diff
@@ -110,13 +112,21 @@ private List<String> getBuildTargetTags(boolean hasTests) {
   }
 
   private void setJvmBuildTarget(GradleSourceSet sourceSet, BuildTarget bt) {
+    DefaultJavaExtension javaExtension = ConversionUtils.toJavaExtension(
+        sourceSet.getExtensions().get("java"));
```

Comment:
It's better to define the supported language id as constants for reuse in model, plugin and server.